SAMtags.tex

```diff
@@ -453,6 +455,106 @@ \subsubsection{Color space}
 Color read quality on the original strand of the read. Same encoding as {\sf QUAL}; same length as {\tt CS}.
 \end{description}
 
+\subsection{Base modifications}
+
+Base basications, including base methylation, are represented as a
+series of edits from the primary unmodified sequence stored in the
+main SAM {\sf SEQ} field.  If the modifed base has no natural
+unmodified form then this should be stored as ``N''.
+
+Each modified base listed also has a quality value associated with
+it.  Given the unmodified base already has a phred likelihood, this
+base modification quality should be interpreted as the likelihood of
+this modification being correct, rather than the base being unmodified.
+
+\begin{description}
+\item[MM:Z:\tagregex{([ACGTN][-+][a-z](,[0-9]+)+;)*}]
+\hfill\\
+The first character is the unmodified base as seen in the {\sf SEQ}
+field, one of {\tt A}, {\tt C}, {\tt G}, {\tt T} or {\tt N}, with
+the exception that {\tt N} is used to match any base rather than
+strictly {\tt N}.  This is followed by either plus or minus
+indicating the strand the modification was observed on (relative to
+the recorded strand of {\sf SEQ} with plus meaning same
+orientation), and a base modification symbol.  This is then followed
+by a comma separated list of how many unmodified seq bases of the
+stated base type to skip, stored as a delta to the last and starting
+with 0 as the first (or next) base.  Hence this number series is
+comparable to the numbers in an {\tt MD} tag.
+
+For example {\tt C+m,5,12,0;} tells us there are three 5-Methylcytosine
+bases in the original {\sf SEQ}.  The first 5 {\tt C} bases are
+unmodified and the 6th is modified, as are the 19th (12 inbetween the
```

Comment:
`in-between` or simply `bases between` seems better (though my english is from 🇮🇱...so who knows?)

Reply:
"Between" makes most sense. Thanks.